frontend/src/components/eMIB/EmailPreview.jsx

```diff
@@ -55,6 +60,15 @@ class EmailPreview extends Component {
     isSelected: PropTypes.bool.isRequired
   };
 
+  // if there is a "(" in the From field, remove it and return
```

Comment:
I would add something here to specify that this applies only for the preview emails, even if we are in _EmailPreview.jsx_ file. This would avoid all confusion.

Reply:
Added details in c009660